api/urls.py

```diff
@@ -4,6 +4,11 @@
 
 urlpatterns = [
     path('banks/', views.BankListAPIView.as_view(), name='banks_api'),
-    path('user-token/', views.CreateUserTokenAPIView.as_view(), name='user_token_api'),
-    path('accounts/', views.AuthorizedUserAccountsAPIView.as_view(), name='user-accounts_api'),
+    path('bank/create/', views.CreateBankAPIView.as_view(), name='create_bank_api'),
+    path('user-token/', views.TokenObtainAPIView.as_view(), name='user_token_api'),
+    path('accounts/', views.AccountListAPIView.as_view(), name='user_accounts_api'),
```

Comment:
for listing, create:  `/accounts/`
for detail, udpate & destory: `/account/12/`